learn/primary_db_sync/meilisync_postgresql.mdx

```diff
@@ -0,0 +1,192 @@
+
+</Tabs.Content>
+
+<Tabs.Content label="Manual install">
+### Manual install
+
+First, make sure you have Python 3.9 and `pip` installed.
+
+Then, open your command-line console and run the following command to install `meilisync`:
+
+```sh
+pip install meilisync
+```
+
+`pip` will install `meilisync`. Once it is done, run the following command to verify your installation:
+
+```sh
+meilisync --help
+```
+
+If the install process was successful, you should see a brief description of `meilisync`'s options.
+
+</Tabs.Content>
+</Tabs.Container>
+
+## Connect `meilisync` to Meilisearch
+
+With `meilisync` installed, you now need to connect it to your Meilisearch instance.
+
+Create a `config.yml` file in your project's directory. Open it with a text editor and add the following configuration options depending on whether you are using Docker or not:
+
+<Tabs.Container labels={["Docker", "Manual install"]}>
+<Tabs.Content label="Docker">
+```yaml
+meilisearch:
+  api_url: http://host.docker.internal:7700/
+  api_key: 'MEILISEARCH_API_KEY'
+  insert_size: 1000
+  insert_interval: 10
+```
+</Tabs.Content>
+
+<Tabs.Content label="Manual install">
+```yaml
+meilisearch:
+  api_url: MEILISEARCH_API_URL
+  api_key: 'MEILISEARCH_API_KEY'
+  insert_size: 1000
+  insert_interval: 10
+```
+</Tabs.Content>
+</Tabs.Container>
+
+Replace `MEILISEARCH_API_KEY` with an API key able to create, update, and delete documents and indexes. Replace `MEILISEARCH_API_URL` with the your instance's API URL. If you are using [Meilisearch Cloud](https://cloud.meilisearch.com), this URL should look similar to this: https://ms-4d85L33tC0d3-5041.fra.meilisearch.io
+
+## Configure PostgreSQL
+
+After connecting `meilisync` to Meilisearch, you need to connect it to your PostgreSQL database.
+
+### Enable logical replication
+
+To use `meilisync` with PostgreSQL, you need to enable the [`wal2json`](https://github.com/eulerto/wal2json) extension.
+
+The `wal2json` extension requires the [Write-Ahead Log settings](https://www.postgresql.org/docs/current/runtime-config-wal.html) to be set to `logical`.
+
+This can be done by adding the following line to your PostgreSQL server's `postgresql.conf` file:
+
+```conf
+wal_level = logical
+#
+# these parameters only need to set in versions 9.4, 9.5 and 9.6
+# default values are ok in version 10 or later
+#
+max_replication_slots = 10
+max_wal_senders = 10
+```
+
+Alternatively, set the binary log format during runtime by running the following command in your PostgreSQL server console:
+
+```sql
+ALTER SYSTEM SET wal_level = logical;
+```
+
+This will require a server configuration reload to take effect. Read more about this in the [`ALTER SYSTEM` documentation](https://www.postgresql.org/docs/current/sql-altersystem.html).
+
+### Connect `meilisync` and PostgreSQL
+
+Open `meilisync`'s `config.yml` once again and append the following settings to the bottom of the file:
+
+```yaml
+source:
+  type: postgres
+  host: 127.0.0.1
```

Comment:
I realize that I put the basic configuration if PostgreSQL is hosted on the same server as meilisync maybe we could add a note that the URL/host and port can be changed depending on the address of the Postgresql database. Not sure if it's mandatory

Reply:
I added a note about this! Great catch, thanks